.gitignore

```diff
@@ -26,10 +26,7 @@ CATcher.iml
 
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
```

Comment:
Not so sure if there is a reason to remove these lines in this PR! ><

Reply:
I asked Anubhav about this, he said I can remove these lines (these files don't exist anymore) but yeah, probably better to remove them in another pr! Will do so

Reply:
Yeee best to do in another PR XD